charts/woodpecker/charts/server/templates/secret.yaml

```diff
@@ -11,3 +11,18 @@ data:
   {{ $key }}: {{ $value | b64enc }}
   {{- end }}
 {{- end }}
+---
+{{- if .Values.createAgentSecret }}
+{{- $secret := lookup "v1" "Secret" .Release.Namespace "woodpecker-default-agent-secret" -}}
+  {{- if or (not $secret) (not (hasKey $secret.data "WOODPECKER_AGENT_SECRET")) }}
+# Secret or key doesn't exist and createAgentSecret: true -> create secret
+apiVersion: v1
+kind: Secret
+metadata:
+  name: woodpecker-default-agent-secret
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+data:
+  WOODPECKER_AGENT_SECRET: {{ randAlphaNum 64 | b64enc }}
```

Comment:
`randAlphaNum` should be secure enough, but was there a reason not to use `randAscii`?